### DISCUSSION TITLE
Questions for installing hrldas/Noahmp

### DISCUSSION THREAD

**Discussion by ghost** [NCAR/noahmp#105] [Q&A]
### I am downloading the NoahMP model, and after entering the 'make all' command, it encounters these errors.

> /usr/include/sys/cdefs.h:31:3: error: #error "You need a ISO C conforming compiler to use the glibc headers"
>  error "You need a ISO C conforming compiler to use the glibc headers"
>    ^~~~~
> make[1]: *** [module_hrldas_netcdf_io.o] Error 1
> make[1]: Leaving directory `/home/zcc/noahmp/hrldas/hrldas/IO_code'
> make: *** [all] Error 2

**Comment by keenmisty** [comment]
You may check the C preprocessor of your gcc
